bessctl/conf/testing/run_module_tests.bess

```diff
@@ -8,6 +8,9 @@ import sugar
 import traceback
 from time import gmtime, strftime
 
+# Env variable used to run a single test
+test_name = $TEST_NAME!'none'
+
```

Comment:
I wonder if it won't be better to just set the default value to `*`, so something like

```
test_name = $TEST_NAME!'*'
```

This has two benefits: one it saves the if block below; two it allows testa named `none.py`.